app/src/main/res/values/strings.xml

```diff
@@ -2,6 +2,13 @@
     <string name="app_name">Books</string>
     <string name="label_thumbnail">Book thumbnail</string>
 
+    <string name="murder_on_the_orient_express">Murder on the Orient Express</string>
+    <string name="lorem_ipsum_001">Lorem ipsum</string>
```

Comment:
te string po co?

Reply:
Po to, że *tools:text* jest widoczne tylko w edytorze, a różne *lorem ipsum* pozwalają mi na podgląd układu *TextView*, w zależności od długości tekstu.

Reply:
ok, ok już wiem i biję się w pierś. Przepraszam. :)

Reply:
😄

Reply:
spoko, ale żeby nie zaśmiecać pliku z tekstami faktycznie widocznymi w aplikacji można to przenieść do osobnego np. lorem.xml i dodać atrybut translatable="false"

Reply:
Dokładnie. Albo po prostu wpisać je z palca w xmlu jeśli są krótkie, AS nie protestuje przeciwko temu 😉